core/shared/src/test/scala/com/lynbrookrobotics/potassium/streams/StreamTest.scala

```diff
@@ -299,4 +299,22 @@ class StreamTest extends FunSuite {
 
     assert(emitted == 3)
   }
-}
+
+  test("Stream of 3 ft minus Stream of 2 ft produces stream of 1") {
+    // minuend is what to subtract from
+    val (minuend, pubMinuend) = Stream.manual[Length]
+
+    // subtractand is what to subtract from minuend
+    val (subtractand, pubSubtractand) = Stream.manual[Length]
+
+    val difference = minuend.minus(subtractand)
```

Comment:
That's a good point, now that I think of it we probably just shouldn't even add the `-` method.